apps/backoffice/src/pages/services/index.tsx

```diff
@@ -454,7 +469,14 @@ export default function Services() {
               passHref
               style={{ textDecoration: "none" }}
             >
-              <Button size="medium" startIcon={<Add />} variant="contained">
+              <Button
+                size="medium"
+                variant="contained"
+                startIcon={<Add />}
+                onClick={() => {
+                  logToMixpanel("IO_BO_SERVICE_CREATE_START", {});
```

Comment:
is it possible to refine the `logToMixpanel` function signature so that it doesn't have to pass empty object if there is nothing to specify?

Reply:
I tried to change the control to make possible to pass nothing if the event doesn't need any prop. but doing this it makes all events to have two kind of behaviour, anyone can accept both nothing or the proper pros that they needs